lib/strainer/behaviors/action_view_image_tag_changes.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'action_view/helpers/asset_tag_helper'
```

Comment:
This will force the action_view component to load out of order. Can you remove this? It should work without.